app/subscription/share.py

```diff
@@ -334,7 +337,8 @@ def process_inbounds_and_tags(
                         "ais": host["allowinsecure"]
                         or inbound.get("allowinsecure", ""),
                         "mux_enable": host["mux_enable"],
-                        "fragment_setting": host["fragment_setting"]
+                        "fragment_setting": host["fragment_setting"],
+                        "mode" : "multi" if inbound.get('multiMode', '') == True else "gun"
```

Comment:
you can just use boolean in here to avoid if in line 515

Reply:
@M03ED im sorry but what is stopping this pr from merging? except the suggestion part - because multi-mode not getting defined here causes ios clients to become unstable so its kinda crucial